### PR TITLE
Rahu eat search api

### PR DESCRIPTION
**Pull request by rahu-eGov**
EAT search API

### CODE REVIEW

File: domain-services/ifix-master-data-service/src/main/java/org/egov/util/GovernmentConstants.java [hunk 1]
```diff
@@ -4,8 +4,12 @@
     public static final String USER_INFO = "USER_INFO";
     public static final String GOVERNMENT_ID = "GOVERNMENT_ID";
     public static final String GOVERNMENT_NAME = "GOVERNMENT_NAME";
+    public static final String TENANT_ID = "TENANT_ID";
```
Comment (rushang7-eGov) [inline domain-services/ifix-master-data-service/src/main/java/org/egov/util/GovernmentConstants.java:7]: If this file is going to include constants apart from Government then change the file name to MasterDataConstants instead of GovernmentConstants. (Do this in a separate PR.

File: domain-services/ifix-master-data-service/src/main/java/org/egov/web/controllers/EatApiController.java [hunk 2]
```diff
@@ -17,25 +18,33 @@
...
         this.objectMapper = objectMapper;
         this.request = request;
     }
 
-    @RequestMapping(value = "/eat/v1/_create", method = RequestMethod.POST)
-    public ResponseEntity<EatResponse> eatV1CreatePost(@ApiParam(value = "Details for the new department + RequestHeader (meta data of the API).", required = true) @Valid @RequestBody EatRequest body) {
+    @RequestMapping(value = "/_create", method = RequestMethod.POST)
+    public ResponseEntity<EatResponse> eatV1CreatePost(@ApiParam(value = "Details for the new department + RequestHeader" +
+            " (meta data of the API).", required = true) @Valid @RequestBody EatRequest body) {
         String accept = request.getHeader("Accept");
```
Comment (rushang7-eGov) [inline domain-services/ifix-master-data-service/src/main/java/org/egov/web/controllers/EatApiController.java:48]: Remove this code and only keep the last line in the method i.e. return response code NOT_IMPLEMENTED.

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

File: domain-services/ifix-master-data-service/src/main/java/org/egov/web/controllers/EatApiController.java (hunk 2 above)
Comment (rahu-eGov) [reply domain-services/ifix-master-data-service/src/main/java/org/egov/web/controllers/EatApiController.java]: Done

File: domain-services/ifix-master-data-service/src/main/java/org/egov/util/GovernmentConstants.java (hunk 1 above)
Comment (rahu-eGov) [reply domain-services/ifix-master-data-service/src/main/java/org/egov/util/GovernmentConstants.java:7]: I have moved constant names in master constant file name.
